client/src/components/forms/SignUp/index.tsx

```diff
@@ -100,7 +103,7 @@ function SignUpForm({
 
   return (
     <div>
-      <Typography variant="h4" align="center">
+      <Typography variant="h4" align="center" className={classes.title}>
```

Comment:
💯

Reply:
thinking about it now i could have just used one of these boys `<em>...</em>`